funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -127,6 +127,7 @@ def __init__(self,
                  logging_level=logging.INFO,
                  endpoint_id=None,
                  suppress_failure=False,
+                 relax_manager_version_match=True,
```

Comment:
i don't like naming of this: relax compared to what? merely to historic behaviour which will soon disappear into the mists of time

it's also a "don't do something = True" flag which is confusing when there are double negatives: eg false means: "don't not compare versions"

this is frustrating from a usability perspective.

a more positive expression of enabling behaviour would be something like: require_manager_endpoint_python_match (or something much shorter?) with inverted bool behaviour compared to this PR.

Reply:
What do you think about removing this option entirely? I think having this option adds little value.

Reply:
yeah I think thats probably fine